### PR TITLE
Add Helix guidelines

### PR DESCRIPTION
**Pull request by retohugi**

### CODE REVIEW

File: content/Helix-Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Helix Guidelines # this is the Title of your Blip
+ring: Adopt # Values: [Hold | Assess | Trial | Adopt]
+quadrant: Techniques # Values: [Tools-And-Libraries | Techniques | Modules | Products] - Make sure to keep these exact values, the Radar is also case sensitive.
+isNew: TRUE # Values: [TRUE | FALSE] - Make sure to keep the value all uppercase.
+---
+
+The [Helix Guidelines](https://helix.sitecore.com/) should not require an introduction anymore. It is the defacto standard for Sitecore developers to structure, architect and build Solutions.
```
Comment (schwindelig) [inline content/Helix-Guidelines.md:8]: `defacto` seems to be a company :) According to [Wikipedia](https://en.wikipedia.org/wiki/De_facto), `de facto` would be correct

**Review by schwindelig — APPROVED** [review]

**Review by ernstjoss — APPROVED** [review]

File: content/Helix-Guidelines.md (hunk 1 above)
Comment (retohugi) [reply content/Helix-Guidelines.md:8]: Latin is hard :) - thanks for your thorough review!

Comment (retohugi) [inline content/Helix-Guidelines.md:8]:
```suggestion
The [Helix Guidelines](https://helix.sitecore.com/) should not require an introduction anymore. It is the de facto standard for Sitecore developers to structure, architect and build Solutions.
```

Comment (retohugi) [reply content/Helix-Guidelines.md:8]: @schwindelig suggestions are also supported with GH reviews :)

**Pull request merged**